emission-factors/activities/activity_based_EF_database_out.csv

```diff
@@ -1,525 +1,525 @@
-Scope 2,Electricity,,,HR,kWh,2020,Scope2_Electricity_HR_kWh_2020,,0.22696,0.4688,AIB,"The Association of Issuing Bodies (AIB) publishes yearly datasets of electricity emission factors taking into account guarantees of origin in the European electricity market. The location-based emission factors are from the production mix per country and year, while the market-based factors are from the residual mix. All factors only take into account the CO2-emissions, not any or the other greenhouse gases. ",https://www.aib-net.org/facts/european-residual-mix/2020,HR_kWh_2020_electricity_BLANK_BLANK
-Scope 2,Electricity,,,HR,kWh,2021,Scope2_Electricity_HR_kWh_2021,,0.20052,0.4665,AIB,"The Association of Issuing Bodies (AIB) publishes yearly datasets of electricity emission factors taking into account guarantees of origin in the European electricity market. The location-based emission factors are from the production mix per country and year, while the market-based factors are from the residual mix. All factors only take into account the CO2-emissions, not any or the other greenhouse gases. ",https://www.aib-net.org/facts/european-residual-mix/2021,HR_kWh_2021_electricity_BLANK_BLANK
-Scope 2,Electricity,,,HU,kWh,2019,Scope2_Electricity_HU_kWh_2019,,0.253,0.286,AIB,"The Association of Issuing Bodies (AIB) publishes yearly datasets of electricity emission factors taking into account guarantees of origin in the European electricity market. The location-based emission factors are from the production mix per country and year, while the market-based factors are from the residual mix. All factors only take into account the CO2-emissions, not any or the other greenhouse gases. ",https://www.aib-net.org/facts/european-residual-mix/2019,HU_kWh_2019_electricity_BLANK_BLANK
-Scope 2,Electricity,,,HU,kWh,2020,Scope2_Electricity_HU_kWh_2020,,0.24375,0.27411,AIB,"The Association of Issuing Bodies (AIB) publishes yearly datasets of electricity emission factors taking into account guarantees of origin in the European electricity market. The location-based emission factors are from the production mix per country and year, while the market-based factors are from the residual mix. All factors only take into account the CO2-emissions, not any or the other greenhouse gases. ",https://www.aib-net.org/facts/european-residual-mix/2020,HU_kWh_2020_electricity_BLANK_BLANK
-Scope 2,Electricity,,,HU,kWh,2021,Scope2_Electricity_HU_kWh_2021,,0.22397,0.27626,AIB,"The Association of Issuing Bodies (AIB) publishes yearly datasets of electricity emission factors taking into account guarantees of origin in the European electricity market. The location-based emission factors are from the producti
```

Comment:
❓ Is this exclluded from the build?

Reply:
Do mean, if it should? We could. Doesn't really make a difference.
The out file is just the same CSV with the key column